Terraform/AWS/task6/Level2/lb.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_lb" "main" {
+  name               = var.env_code
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.lb_allow_http_public.id]
+  subnets            = data.terraform_remote_state.level1.outputs.public_subnet
+
+  tags = {
+    Environment = var.env_code
+
+  }
+}
+resource "aws_security_group" "lb_allow_http_public" {
+  name        = "lb_allow_tls"
+  description = "Allow TLS inbound traffic"
+  vpc_id      = data.terraform_remote_state.level1.outputs.vpc_id
+
+  ingress {
+    description = "http from my IP"
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 65535
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "${var.env_code}-allow_http-from-IP"
+  }
+}
+resource "aws_lb_target_group" "main" {
+  name     = var.env_code
+  port     = 80
+  protocol = "HTTP"
+  vpc_id   = data.terraform_remote_state.level1.outputs.vpc_id
+
+  health_check {
+    enabled             = true
+    path                = "/"
+    port                = "traffic-port"
+    healthy_threshold   = 5
+    unhealthy_threshold = 2
+    timeout             = 5
+    interval            = 30
+    matcher             = 200
+  }
+}
+resource "aws_lb_target_group_attachment" "main" {

```

Comment:
you should have 2 `aws_lb_target_group_attachment` and associate the two aws private instance to it